learn/features/treasury.md

```diff
@@ -1,79 +1,33 @@
-    | Target % of transaction fees allocated to Treasury |                                     {{ networks.moonbase.treasury.tx_fees_allocated }}                                      |
-
---8<-- 'text/_common/async-backing.md'
-
-## Community Treasury {: #community-treasury }
-
-The Treasury is funded by a percentage of each block's transaction fees. The remaining percentage of the fees is burned (check the table above). The Treasury allows stakeholders to submit spending proposals to be reviewed and voted on by the Treasury Council. These spending proposals should include initiatives to further the network or boost network engagement. Some network initiatives could include funding integrations or collaborations, community events, network outreach, and more. Before a proposal is submitted, the author of the proposal can submit their idea for their proposal to the designated Treasury [discussion forum](https://forum.moonbeam.network){target=\_blank} for feedback from the community for at least five days.
-
-To deter spam, proposals must be submitted with a deposit, also known as a proposal bond. The proposal bond is a percentage (check the table above) of the amount requested by the proposer, with a minimum amount and no upper limit (compared to Polkadot and Kusama, which have a maximum bond amount). A governance proposal can change these values. So, any token holder with enough tokens to cover the deposit can submit a proposal. If the proposer doesn't have enough funds to cover the deposit, the extrinsic will fail due to insufficient funds, but transaction fees will still be deducted.
-
-Once a proposal has been submitted, a Treasury Council member may motion for a vote on the proposal. The Treasury Council then votes on it during the motion duration. The threshold for accepting a treasury proposal is at least three-fifths of the treasury council. On the other hand, the threshold for rejecting a proposal is at least one-half of the treasury council. If any member(s) of the Treasury Council fails to vote during the motion duration, the vote of the Treasury Council member that holds the "Default Vote" position acts as the default. The "Default Vote" position mirrors that of [Polkadot's "Prime Member"](https://wiki.polkadot.network/docs/learn/learn-governance#prime-members){target=\_blank}. Please note that there is no way for a user to revoke a treasury proposal after it has been submitted.
 
-If approved by the treasury council, the proposal enters a queue to be placed into a spend period. If the spending queue happens to contain the number of maximum approved proposals, the proposal submission will fail similarly to how it would if the proposer's balance is too low. If the proposal gets rejected, the deposit is non-refundable.
+--8<-- 'text/learn/features/treasury/treasury-addresses-path.md'
```

Comment:
I don't think it makes much sense to have the addresses under "General Definitions"